drafts/Stormless3.md

```diff
@@ -0,0 +1,322 @@
+
+# Episode 3
+
+
+Build a serverless app on AWS with [Serverless Framework][sf], AWS Lambda, StepFunctions, DynamoDB, API Gateway, and ready-to-use functions from StackStorm Exchange open-source catalog.
+
+[sf]: https://serverless.com/framework
+
+You are watching the Episode Three. 
+
+In the [Episode One][ep-one], I described the application we are building, walked you over setting up the development environment and creating Serverless project, and shown how to build a first Lambda function from [StackStorm Exchange][st2-ex] action with [Serverless Framework][sf].
+
+In the [Episode Two][ep-two], we added more actions: one native Lambda to record user info to DynamoDB, and another one from StackStorm Exchange to make a call to ActiveCampaign CRM system. You learned more of `serverless.yml` syntax and practiced developer workflow working with individual functions. 
+
+[ep-one]:https://medium.com/@dzimine/tutorial-building-a-community-on-boarding-app-with-serverless-stepfunctions-and-stackstorm-b2f7cf2cc419
+
+[ep-two]: https://medium.com/@dzimine/building-community-sign-up-app-with-serverless-stepfunctions-and-stackstorm-exchange-episode-2-b1efeb1b9bd6
+[st2-ex]: https://exchange.stackstorm.org 
+
+In this episode, I'll show how to use AWS StepFunction to wire the actions into a workflow.
```

Comment:
Little nitpick but this is a brand name, so technically "AWS Step Functions" is correct.